### DISCUSSION TITLE
Laravel Installation

### DISCUSSION THREAD

**Discussion by sahdoio** [opencodeco/.github#4] [Q&A] [answered]
Hi all, nice work here! Really loving the experience!

I'm trying to use laravel default installation. Any tips?
 
![image](https://github.com/opencodeco/phpctl/assets/6379921/0b1477ee-0a1e-4a03-9870-7b7e35e76e00)

**Comment by leocavalcante** [comment]
Hm, it is trying to use global Composer binaries. Didn't thought about those cases.
I'll be working on that.

**Comment by leocavalcante** [comment, marked as answer]
Hi, @sahdoio. Just fixed it. Can you make a new attempt?

Please, run:
```shell
phpctl self-update
```
To make sure you are in the latest update that contains the fix.

Then:
```shell
phpctl composer global require laravel/installer
```

And retry:
```shell
laravel
```
<img width="1025" alt="Screenshot 2024-01-31 at 18 51 58" src="https://github.com/opencodeco/phpctl/assets/183722/1442fd6f-e45d-4d47-a081-45a5932bb2a1">

**Reply by sahdoio** [reply]
Omg! Beautiful! Huge thanks!! 

![image](https://github.com/opencodeco/phpctl/assets/6379921/cfc69437-8688-4e83-ac0b-b9e5c2ac24de)

**Comment by leocavalcante** [comment]
Awesome! Feel free to open any other questions, give some suggestions or issues.
